### PR TITLE
Add an API to change net namespace of RDMA device

### PR DESCRIPTION
**Pull request by paravmellanox**
Enrich netlink package by adding an API to change net namespace of RDMA device similar to

$ rdma dev set [DEV] netns NSNAME

Signed-off-by: Parav Pandit <parav@mellanox.com>

Test results on local system:
go test -v -run TestRdmaLinkSetNsFd
=== RUN   TestRdmaLinkSetNsFd
--- PASS: TestRdmaLinkSetNsFd (0.00s)
    rdma_link_test.go:112: current rdma netns mode exclusive
    rdma_link_test.go:139: rdma link:  &{{12 foo 16.25.1000 12:d1:b3:00:03:07:8a:24 12:d1:b3:00:03:07:8a:24}}
PASS
ok      github.com/vishvananda/netlink  0.009s

### CODE REVIEW

File: rdma_link_linux.go [hunk 1]
```diff
@@ -238,3 +238,27 @@ func (h *Handle) RdmaSystemSetNetnsMode(NewMode string) error {
...
+// Similar to: `rdma dev set $dev netns $ns`
+func (h *Handle) RdmaLinkSetNsFd(link *RdmaLink, fd uint32) error {
+	proto := getProtoField(nl.RDMA_NL_NLDEV, nl.RDMA_NLDEV_CMD_SET)
+	req := h.newNetlinkRequest(proto, unix.NLM_F_ACK)
+
+	data := nl.NewRtAttr(nl.RDMA_NLDEV_ATTR_DEV_INDEX,
+		nl.Uint32Attr(uint32(link.Attrs.Index)))
+	req.AddData(data)
+
+	data = nl.NewRtAttr(nl.RDMA_NLDEV_NET_NS_FD, nl.Uint32Attr(uint32(fd)))
```
Comment (paravmellanox) [inline rdma_link_linux.go]: fd is actually already uint32. I don't need to typecast.
let me fix it.

**Comment by paravmellanox** [comment]
Hi @aboch 

Can you please review this short API addition?

**Comment by aboch** [comment]
LGTM

**Comment by paravmellanox** [comment]
Thanks @aboch

**Pull request merged**